### PR TITLE
Merge actual state of project in Master branch

### LINKED ISSUE #20: Merge actual state of project in Master branch
I think it no longer makes sense to work at the resource branch.
In my opinion, we should migrate the current state of this repository to master to work in the same pattern as other projects.

### PR DESCRIPTION
**Pull request by marcellanz**
fixes #20 "Merge actual state of project in Master branch".

I'll capture the information about how to start a language support lib, whihc I think was quite valuable, somewhere else.

I also removed https://lgtm.com and travis badges, as they where outdated with the new repo. Regarding CI, I tried just shortly Github Actions. 
WDYT @sleipnir @chomnoue is it worth to try Github Actions or should we stick with travis?

### CODE REVIEW

**Comment by marcellanz** [comment]
@pvlugter can you change the default branch from currently 'feature/python-support' to 'master' on the repos Settings page? Thanks.

**Comment by sleipnir** [comment]
I believe that all Cloudstate repositories today are using Travis CI, I have nothing against "Actions" but I think we should keep travis for compliance. But it's a good question for Lightbend people to answer

File: build/compile-protbuf.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```
Comment (sleipnir) [inline build/compile-protbuf.sh:1]: Should we include this in Travis' pipeline?

File: build/fetch-cloudstate-pb.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```
Comment (sleipnir) [inline build/fetch-cloudstate-pb.sh:1]: Should we include this in Travis' pipeline?

File: cloudstate/cloud_state.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        self.event_sourced_entities.append(entity)
+        return self
+
+    def start(self):
+        server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+        add_EntityDiscoveryServicer_to_server(CloudStateEntityDiscoveryServicer(self.event_sourced_entities), server)
+        add_EventSourcedServicer_to_server(CloudStateEventSourcedServicer(self.event_sourced_entities),server)
+        port = os.environ.get('HOST', '127.0.0.1') + ':' + os.environ.get('PORT', '8080')
+        server.add_insecure_port(port)
+        pprint('Starting Cloudstate on ' + port)
```
Comment (sleipnir) [inline cloudstate/cloud_state.py:31]: I think we could replace this with the standard logging library

File: cloudstate/discovery_servicer.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        )
+        descriptor_set.file.append(
+            FileDescriptorProto.FromString(Default().FindFileByName('google/api/http.proto').serialized_pb)
+        )
+        spec = entity_pb2.EntitySpec(
+            service_info=entity_pb2.ServiceInfo(
+                service_version='0.1.0',
+                service_runtime='Python ' + platform.python_version() + ' [' + platform.python_implementation() + ' ' +
+                                platform.python_compiler() + ']',
+                support_library_name='cloudstate-python-support',
```
Comment (sleipnir) [inline cloudstate/discovery_servicer.py:58]: Rename to python-support

File: cloudstate/discovery_servicer.py [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                support_library_version='0.1.0'
```
Comment (sleipnir) [inline cloudstate/discovery_servicer.py:59]: I think we should get the version in an externalized way (get it from a file maybe or via setuptools). Maybe using pkg_resources

File: cloudstate/evensourced_servicer.py [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+    return cmd
+
+
+def pack(event):
+    any = Any()
+    any.Pack(event)
+    return any
+
+
+class CloudStateEventSourcedServicer(EventSourcedServicer):
```
Comment (sleipnir) [inline cloudstate/evensourced_servicer.py:37]: How about we rename it to CloudstateEventSourcedService?

File: cloudstate/evensourced_servicer.py [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+            elif request.HasField('event'):
+                event:EventSourcedEvent = request.event
+                evt = get_payload(event)
+                event_result = handler.handle_event(current_state, evt,
+                                                    EventContext(entity_id, event.sequence))
+                start_sequence_number = event.sequence
+                if event_result:
+                    current_state = event_result
+                pprint("Handling event {}".format(event))
```
Comment (sleipnir) [inline cloudstate/evensourced_servicer.py:78]: use standart logging here instead pprint

File: cloudstate/tests/test_cloudstate.py [hunk 8]
```diff
@@ -0,0 +1,7 @@
+# content of test_sample.py
+def inc(x):
+    return x + 1
+
+
+def test_answer():
```
Comment (sleipnir) [inline cloudstate/tests/test_cloudstate.py:6]: I need to resolve this here. If anyone has an idea for a simple test feel free to include

**Review by sleipnir — CHANGES_REQUESTED** [review]
Hi guys, I made a few comments

**Comment by pvlugter** [comment]
> @pvlugter can you change the default branch from currently 'feature/python-support' to 'master' on the repos Settings page?

@marcellanz done

**Comment by marcellanz** [comment]
@sleipnir thanks for your review and comment. I agree on them. How about to chamge these things after having this branch on master again. Peter changed default branch back again to "master", so we should have something on it with the progress we had on this feature branch. Lets iterate then on master instead on this PR.

File: build/compile-protbuf.sh (hunk 1 above)
Comment (marcellanz) [reply build/compile-protbuf.sh:1]: should/do we compile then on demand like in the java lib?

File: build/fetch-cloudstate-pb.sh (hunk 2 above)
Comment (marcellanz) [reply build/fetch-cloudstate-pb.sh:1]: same comment as fr compile protobuf script.

File: cloudstate/cloud_state.py (hunk 3 above)
Comment (marcellanz) [reply cloudstate/cloud_state.py:31]: Agree, lets have an issue for that and have logging story that makes sense.

File: cloudstate/discovery_servicer.py (hunk 4 above)
Comment (marcellanz) [reply cloudstate/discovery_servicer.py:58]: Agree.

File: cloudstate/discovery_servicer.py (hunk 5 above)
Comment (marcellanz) [reply cloudstate/discovery_servicer.py:59]: great point, Agree.

File: cloudstate/evensourced_servicer.py (hunk 6 above)
Comment (marcellanz) [reply cloudstate/evensourced_servicer.py:37]: yeah, makes sense since LB renamed it tha way 👍

**Pull request merged**